src/README.md

```diff
@@ -0,0 +1 @@
+# LazyLedger Specifications
```

Comment:
This looks a bit too empty. Can't we show the summary here instead?
![image](https://user-images.githubusercontent.com/7441766/120081699-2bbc9d80-c0bf-11eb-81aa-4598304fbfe7.png)

Reply:
I just did a migration without adding any content that didn't need to be added for the structure of the book. But sure, I can add ToC for each chapter and the top level. Ideally it would eventually contain more high-level guidance.

Reply:
Fixed in a3be997.

Reply:
Yeah, makes sense. We should do this before we deploy them though (and we should start deploying the spec asap).